package.json

```diff
@@ -1154,6 +1155,22 @@
           "description": "Specify a shell command that is run to start the browser for authenticating with the OIDC identity provider for the server connection. Leave this empty for default browser."
         }
       }
+    },
+    "icons": {
+      "connection-active": {
```

Comment:
Should we prefix this with a MongoDB identifier? That way if there's ever a `connection-active` added to the long icon list https://code.visualstudio.com/api/references/icons-in-labels#icon-listing or by another extension we don't conflict.
Maybe `mongodb-connection-active`?

Reply:
agreed, it'd actually help with the JSON schema for this as VSCode always expects at least one `-`. Could `mdb-` be alright too to keep the strings with icons short?

Reply:
`mdb-` sgtm!